---
layout: default
---



<section>
    <div>
        <p>
        <img  width="50" height="50" alt="icon of a protein" src="./images/protein-icon_50x50.png"/ ><strong>Protein Function Prediction</strong><br>
        A major interest of ours is the prediction of protein function. Genomics, proteomics and various other ``-omics'' 
        inundate us with sequence and structure information, but the biological functions of those proteins in many cases still eludes us. 
        Computational prediction of protein and gene function is a rapidly growing research field in bioinformatics 
        [<a href="https://www.ncbi.nlm.nih.gov/pubmed/16772267?dopt=AbstractPlus">1</a>]. 
        Our lab co-organizes the automated computational protein function prediction meetings: <a href="http://biofunctionprediction.org/">AFP</a>. 
        The AFP meetings bring together researchers to discuss various methods for protein function prediction. 
        We also co-organize the <a href="http://biofunctionprediction.org/CAFA">Critical Assessment of Function Annotation</a>, or CAFA. 
        <a href="http://biofunctionprediction.org/CAFA">CAFA</a> is an experiment designed to provide a large-scale assessment of computational methods dedicated to          
        predicting protein function. Different algorithms are evaluated by their ability to predict the Gene Ontology (GO) terms in 
        the categories of Molecular Function, Biological Process, and Cellular Component. The <a href="https://www.kaggle.com/competitions/cafa-5-protein-function-prediction">
        latest iteration of CAFA</a> took place on the Kaggle website, with a record number of nearly 
        2,000 participants. 
    </div>
</p>
    <p>
<div>
        <strong>NEW (2025-8-1):</strong> We are recruiting PhD students to work on the next CAFA round. This is an exciting opportunity to engage with internationally leading research groups in 
        computational protein function prediction and to contribute to the solution of this important problem. See the <a href="https://friedberglab.net/contact">Contact page</a>a> 
        for contact details. 
        Please put "[CAFA PhD]" in your email subject and include a CV and a brief cover letter with a statement of interest  
           
     </div>  
     </p>
    
    
</section>

<section >
          
    

    <div>
        <img width="50" height="50" alt="icon of an evolutionary tree" src="./images/noun-evolutionary-tree-4x4.png"/>               
        <strong>Bacterial Genome Evolution</strong> <br>Gene blocks are a common occurrence in bacteria: these are genes that lie close together on the chromosome and may participate in a common cellular or biochemical function. Operons are gene blocks whose member genes are co-transcribed. We have developed a new method to describe the evolution of operons and gene blocks in bacteria. We describe a small set of evolutionary events that can take place in gene block evolution, and count these events to create a new type of molecular clock that tells us <a href="https://academic.oup.com/bioinformatics/article/31/13/2075/196486">how fast or how slow certain gene blocks may have evolved</a>. We hope to learn how new funcitons are acquired by ensembles of genes such as these.</p>
    </div>
              
</section>

<section>
    
     <div>
      <img width="50" height="50" alt="icon of a metagenome" src="./images/metagenome-50x50.png "/>
      <strong>Metagenomics</strong><br/> Metagenomics is the study of genomic material 
      extracted directly from the environment. New sequencing technologies have enabled the study of whole populations of 
       genomes taken from microbial communities in the field, as opposed to single-species clonal cultures in the lab. 
       Metagenomics offers a way to study how genomes evolve to cope with the microbial biotic and abiotic environments. 
         We are also looking at horizontal gene transfer as a driver of evolution in bacteria, especially for antimicrobial resistance.
     </div>
     
</section>

<section>
    
     <div>
      <img width="50" height="50" alt="icon of an antibody" src="./images/antibody-50x50.png"/>
      <strong>Epitope Tagging</strong>Epitope tagging is an invaluable technique enabling the identification, tracking, and purification of proteins in vivo.
         We developed a tool, EpicTope, to facilitate this method by identifying amino acid positions suitable for epitope insertion. Our method uses a scoring 
         function that considers multiple protein sequences and structural features to determine locations least disruptive to the proteinâ€™s function. 
         We validated our approach on the zebrafish Smad5 protein, showing that multiple predicted internally 
         tagged Smad5 proteins rescue zebrafish smad5 mutant embryos, while the N- and C-terminal tagged variants do not, also as predicted.
         
         <br/> 
     </div>
     
</section>


<!--
<div class="posts">
  {% for post in site.posts %}
    <article class="post">

      <h1><a href="{{ site.baseurl }}{{ post.url }}">{{ post.title }}</a></h1>

      <div class="entry">
        {{ post.excerpt }}
      </div>

      <a href="{{ site.baseurl }}{{ post.url }}" class="read-more">Read More</a>
    </article>
  {% endfor %}
</div>
-->

