---
layout: default
---



<section class="mbr-section content6 cid-qJt0qmmcVj" id="content6-2e">
    
     
    
    <div class="container">
        <div class="media-container-row">
            <div class="col-12 col-md-8">
                <div class="media-container-row">
                    <div class="mbr-figure" style="width: 100%;">
                      <a href="https://www.nature.com/articles/nmeth.2340"><img src="assets/images/3-new.jpg" alt="Mobirise" title=""></a>  
                    </div>
                    <div class="media-content">
                        <div class="mbr-section-text">
                            <p class="mbr-text mb-0 mbr-fonts-style display-7"><strong>Function Prediction
</strong><br>A major interest of ours is the prediction of protein function. Genomics, proteomics and various other ``-omics'' inundate us with sequence and structure information, but the biological functions of those proteins in many cases still eludes us. Computational prediction of protein and gene function is a rapidly growing research field in bioinformatics [<a href="https://www.ncbi.nlm.nih.gov/pubmed/16772267?dopt=AbstractPlus">1</a>]. I am the co-organizer of the automated computational protein function prediction meetings: <a href="http://biofunctionprediction.org/">AFP</a>. The AFP meetings bring together researchers to discuss various methods for protein function prediction. My personal interest in function prediction lies in predicting function from protein structure [<a href="https://www.sciencedirect.com/science/article/pii/S0969212607001049?via%3Dihub">2</a>]. We have recently started work on predicting gene function based on its genomic context in bacteria, using both genomic and <a href="http://www.nature.com/nrmicro/focus/metagenomics/index.html">metagenomic </a>data towards that end.</p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </div>
</section>

<section class="mbr-section content7 cid-qJt0mG873o" id="content7-2d">
          
    

    <div class="container">
        <div class="media-container-row">
            <div class="col-12 col-md-8">
                <div class="media-container-row">
                    <div class="media-content">
                        <div class="mbr-section-text">
                            <p class="mbr-text align-right mb-0 mbr-fonts-style display-7"><strong>Bacterial Genome Evolution:</strong> <br>Gene blocks are a common occurrence in bacteria: these are genes which lie close together on the chromosome, and may participate in a common cellular or biochemical function. Operons are gene blocks whose member genes are co-transcribed. We have developed a new method to describe the evolution of operons and gene blocks in bacteria. We describe a small set of evolutionary events that can take place in gene block evolution, and count these events to create a new type of molecular clock that tells us <a href="https://academic.oup.com/bioinformatics/article/31/13/2075/196486">how fast or how slow certain gene blocks may have evolved</a>. We hope to learn how new funcitons are acquired by ensembles of genes such as these.</p>
                        </div>
                    </div>
                    <div class="mbr-figure" style="width: 100%;">
                     <a href="https://www.biorxiv.org/content/early/2017/11/01/212886"><img src="assets/images/2-new.png" alt="Mobirise" title=""></a>  
                    </div>
                </div>
            </div>
        </div>
    </div>
</section>

<section class="mbr-section content6 cid-qJt0lDCSxt" id="content6-2c">
    
     
    
    <div class="container">
        <div class="media-container-row">
            <div class="col-12 col-md-8">
                <div class="media-container-row">
                    <div class="mbr-figure" style="width: 100%;">
                      <a href="https://www.the-scientist.com/August2014/co-occurrence_full1.jpg"><img src="assets/images/1.jpg" alt="Mobirise" title=""></a>  
                    </div>
                    <div class="media-content">
                        <div>
                            <p class="mbr-text mb-0 mbr-fonts-style display-7"><strong>Metagenomics</strong> is the study of genomic material extracted directly from the environment. New sequencing technologies have enabled the study of whole populations of genomes taken from microbial communities in the field, as opposed to single species clonal cultures in the lab. Metagenomics offers a way to study how genomes evolve to cope with the microbial biotic and abiotic environments. Our lab helped developed a method to study the correlation between the human gut microbiota and gut gene expression. We are applying this method towards studying infant gut development the effect of gut microbes on human health and wellness.
<br>
<br></p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </div>




<div class="posts">
  {% for post in site.posts %}
    <article class="post">

      <h1><a href="{{ site.baseurl }}{{ post.url }}">{{ post.title }}</a></h1>

      <div class="entry">
        {{ post.excerpt }}
      </div>

      <a href="{{ site.baseurl }}{{ post.url }}" class="read-more">Read More</a>
    </article>
  {% endfor %}
</div>

