---
layout: page
title: Publications
permalink: /publications/
---

### Lab Publications
For a full publication list, see [Google Scholar](https://scholar.google.com/citations?hl=en&user=0NEtdwoAAAAJ&view_op=list_works&sortby=pubdate)

#### Preprints
+  Zinski J, Chung H, Joshi P, Warrick F, Berg BD, Glova G, McGrail M, Balciunas D, **Friedberg I**, Mullins M EpicTope: narrating protein sequence features to identify non-disruptive epitope tagging sites (2024) _bioRxiv_ 2024.03.03.583232; [doi](https://doi.org/10.1101/2024.03.03.583232)



#### Peer reviewed

+ Banerjee P, Eulenstein O, **Friedberg I** Discovering genomic islands in unannotated bacterial genomes using sequence embedding (2024) _Bioinformatics Advances_ **4**:1 vbae089  [doi](https://doi.org/10.1093/bioadv/vbae089)

+ Piovesan D, Zago D, Joshi P, De Paolis Kaluza MC, Mehdiabadi M, Ramola R, Monzon AM, Reade W, **Friedberg I**, Radivojac P, Tosatto SCE CAFA-evaluator: A Python Tool for Benchmarking Ontological Classification Methods (2024) _Bioinformatics Advances_ vbae043, [doi](https://doi.org/10.1093/bioadv/vbae043)

+ The Critical Assessment of Genome Interpretation Consortium. CAGI, the Critical Assessment of Genome Interpretation, establishes progress and prospects for computational genetic variant interpretation methods. (2024) _Genome Biol_ 25, 53  [doi](https://doi.org/10.1186/s13059-023-03113-6)

+ Mahlich Y, Zhu C, Chung H, Velaga PV, De Paolis Kaluza MC, Radivojac P, **Friedberg I**, Bromberg Y Learning from the unknown: exploring the range of bacterial functionality _Nucleic Acids Research_, (2023) 51:19, 10162â€“10175  [doi](https://doi.org/10.1093/nar/gkad757)
  
+  Chung HC, Foxx CL, Hicks JA, Stuber TP, **Friedberg I**, Dorman KS, Harris B An accurate and interpretable model for antimicrobial resistance in pathogenic Escherichia coli from livestock and companion animal species (2023)  _PLoS ONE_ 18:8 e0290473 [doi](https://doi.org/10.1371/journal.pone.0290473)

+   Joshi P, Banerjee S, Hu X, Khade PM, **Friedberg I** GOThresher: a program to remove annotation biases from protein function annotation datasets _Bioinformatics_ (2023) 39:1 btad048 [doi](https://doi.org/10.1093/bioinformatics/btad048)

+ Ramola R, **Friedberg I**, Radivojac P The field of protein function prediction as viewed by different domain scientists  _Bioinformatics Advances_ 2:1 vbac057 (2022) [doi](https://doi.org/10.1093/bioadv/vbac057)

+ de Crecy-Lagard V, Amorin de Hegedus R, Arighi C, Babor J, Bateman A, Blaby I, Blaby-Haas C, Bridge AJ, Burley SK, Cleveland S, Colwell LJ, Conesa A, Dallago C, Danchin A, de Waard A, Deutschbauer A, Dias R, Ding Y, Fang G, **Friedberg I**, Gerlt J, Goldford J, Gorelik M, Gyori BM, Henry C, Hutinet G, Jaroch M, Karp PD, Kondratova L, Lu Z, Marchler-Bauer A, Martin MJ, McWhite C, Moghe GD, Monaghan P, Morgat A, Mungall CJ, Natale DA, Nelson WC, O'Donoghue S, Orengo C, O'Toole KH, Radivojac P, Reed C, Roberts RJ, Rodionov D, Rodionova IA, Rudolf JD, Saleh L, Sheynkman G, Thibaud-Nissen F, Thomas PD, Uetz P, Vallenet D, Carter EW, Weigele PR, Wood V, Wood-Charlson EM, Xu J
A roadmap for the functional annotation of protein families: a community perspective.
_Database (OUP)_. 2022 Aug 12;2022. pii: 6663924. [doi](https://dx.doi.org/10.1093/database/baac062/)


+ Wymore Brand M, Proctor AL, Hostetter JM, Zhou N, **Friedberg I**, Jergens AE, Phillips GJ, Wannemuehler MJ
Vertical transmission of attaching and invasive E. coli from the dam to neonatal mice predisposes to more severe colitis following exposure to a colitic insult later in life.
_PLoS One_. 2022 Apr 5;17(4):e0266005. [doi](https://dx.doi.org/10.1371/journal.pone.0266005)


+ Kilpatrick AM, Rahman F, Anjum A, Shome S, Andalib KMS, Banik S, Chowdhury SF, Coombe P, Astroz YC, Douglas JM, Eranti P, Kiran AD, Kumar S, Lim H, Lorenzi V, Lubiana T, Mahmud S, Puche R, Rybarczyk A, Al Sium SM, Twesigomwe D, Zok T, Orengo CA, **Friedberg I**, Kelso JF, Welch L
Characterizing domain-specific open educational resources by linking ISCB Communities of Special Interest to Wikipedia.
_Bioinformatics_. 2022 Jun 24;38(Suppl 1):i19-i27. [doi](https://dx.doi.org/10.1093/bioinformatics/btac236)

+ Ramsey J, McIntosh B, Renfro D, Aleksander SA, LaBonte S, Ross C, Zweifel AE, Liles N, Farrar S, Gill JJ, Erill I, Ades S, Berardini TZ, Bennett JA, Brady S, Britton R, Carbon S, Caruso SM, Clements D, Dalia R, Defelice M, Doyle EL, **Friedberg I**, Gurney SMR, Hughes L, Johnson A, Kowalski JM, Li D, Lovering RC, Mans TL, McCarthy F, Moore SD, Murphy R, Paustian TD, Perdue S, Peterson CN, Pruss BM, Saha MS, Sheehy RR, Tansey JT, Temple L, Thorman AW, Trevino S, Vollmer AC, Walbot V, Willey J, Siegele DA, Hu JC
Crowdsourcing biocuration: The Community Assessment of Community Annotation with Ontologies (CACAO).
_PLoS Comput Biol_. 2021 Oct 28;17(10):e1009463. [doi](https://dx.doi.org/10.1371/journal.pcbi.1009463)


+ Martinez-Galvez G, Joshi P, **Friedberg I**, Manduca A, Ekker SC
Deploying MMEJ using MENdel in precision gene editing applications for gene therapy and functional genomics.
_Nucleic Acids Res_. 2021 Jan 11;49(1):67-78. [doi](https://dx.doi.org/10.1093/nar/gkaa1156)


+ Mangravite L, Mooney SD, **Friedberg I**, Guinney J
Establishing the reliability of algorithms.
_Pac Symp Biocomput_. 2021;26:341-345.

+ Nguyen HN, Markin A, **Friedberg I**, Eulenstein O
Finding orthologous gene blocks in bacteria: the computational hardness of the problem and novel methods to address it.
_Bioinformatics_. 2020 Dec 30;36(Suppl_2):i668-i674. [doi](https://dx.doi.org/10.1093/bioinformatics/btaa794)


+ Nett RS, Nguyen H, Nagel R, Marcassa A, Charles TC, **Friedberg I**, Peters RJ
Unraveling a Tangled Skein: Evolutionary Analysis of the Bacterial Gibberellin Biosynthetic Operon.
_mSphere_. 2020 Jun 3;5(3). pii: 5/3/e00292-20. [doi](https://dx.doi.org/10.1128/mSphere.00292-20)



+ Fields FR, Freed SD, Carothers KE, Hamid MN, Hammers DE, Ross JN, Kalwajtys VR, Gonzalez AJ, Hildreth AD, **Friedberg I**, Lee SW
Novel antimicrobial peptide discovery using machine learning and biophysical selection of minimal bacteriocin domains.
_Drug Dev Res_. 2020 Feb;81(1):43-51. [doi](https://dx.doi.org/10.1002/ddr.21601)


+ Zhou N, Jiang Y, Bergquist TR, Lee AJ, Kacsoh BZ, Crocker AW, Lewis KA, Georghiou G, Nguyen HN, Hamid MN, Davis L, Dogan T, Atalay V, Rifaioglu AS, Dalkiran A, Cetin Atalay R, Zhang C, Hurto RL, Freddolino PL, Zhang Y, Bhat P, Supek F, Fernandez JM, Gemovic B, Perovic VR, Davidovic RS, Sumonja N, Veljkovic N, Asgari E, Mofrad MRK, Profiti G, Savojardo C, Martelli PL, Casadio R, Boecker F, Schoof H, Kahanda I, Thurlby N, McHardy AC, Renaux A, Saidi R, Gough J, Freitas AA, Antczak M, Fabris F, Wass MN, Hou J, Cheng J, Wang Z, Romero AE, Paccanaro A, Yang H, Goldberg T, Zhao C, Holm L, Toronen P, Medlar AJ, Zosa E, Borukhov I, Novikov I, Wilkins A, Lichtarge O, Chi PH, Tseng WC, Linial M, Rose PW, Dessimoz C, Vidulin V, Dzeroski S, Sillitoe I, Das S, Lees JG, Jones DT, Wan C, Cozzetto D, Fa R, Torres M, Warwick Vesztrocy A, Rodriguez JM, Tress ML, Frasca M, Notaro M, Grossi G, Petrini A, Re M, Valentini G, Mesiti M, Roche DB, Reeb J, Ritchie DW, Aridhi S, Alborzi SZ, Devignes MD, Koo DCE, Bonneau R, Gligorijevic V, Barot M, Fang H, Toppo S, Lavezzo E, Falda M, Berselli M, Tosatto SCE, Carraro M, Piovesan D, Ur Rehman H, Mao Q, Zhang S, Vucetic S, Black GS, Jo D, Suh E, Dayton JB, Larsen DJ, Omdahl AR, McGuffin LJ, Brackenridge DA, Babbitt PC, Yunes JM, Fontana P, Zhang F, Zhu S, You R, Zhang Z, Dai S, Yao S, Tian W, Cao R, Chandler C, Amezola M, Johnson D, Chang JM, Liao WH, Liu YW, Pascarelli S, Frank Y, Hoehndorf R, Kulmanov M, Boudellioua I, Politano G, Di Carlo S, Benso A, Hakala K, Ginter F, Mehryary F, Kaewphan S, Bjorne J, Moen H, Tolvanen MEE, Salakoski T, Kihara D, Jain A, Smuc T, Altenhoff A, Ben-Hur A, Rost B, Brenner SE, Orengo CA, Jeffery CJ, Bosco G, Hogan DA, Martin MJ, O'Donovan C, Mooney SD, Greene CS, Radivojac P, **Friedberg I**
The CAFA challenge reports improved protein function prediction and new functional annotations for hundreds of genes through experimental screens.
_Genome Biol_. 2019 Nov 19;20(1):244. [doi](https://dx.doi.org/10.1186/s13059-019-1835-8)


+ Wierson WA, Simone BW, WareJoncas Z, Mann C, Welker JM, Kar B, Emch MJ, **Friedberg I**, Gendron WAC, Barry MA, Clark KJ, Dobbs DL, McGrail MA, Ekker SC, Essner JJ
Expanding the CRISPR Toolbox with ErCas12a in Zebrafish and Human Cells.
CRISPR J. 2019 Dec;2(6):417-433. [doi](https://dx.doi.org/10.1089/crispr.2019.0026)


+ Nguyen HN, Jain A, Eulenstein O, **Friedberg I**
Tracing the ancestry of operons in bacteria.
Bioinformatics. 2019 Sep 1;35(17):2998-3004. [doi](https://dx.doi.org/10.1093/bioinformatics/btz053)


+ Kasak L, Bakolitsa C, Hu Z, Yu C, Rine J, Dimster-Denk DF, Pandey G, De Baets G, Bromberg Y, Cao C, Capriotti E, Casadio R, Van Durme J, Giollo M, Karchin R, Katsonis P, Leonardi E, Lichtarge O, Martelli PL, Masica D, Mooney SD, Olatubosun A, Radivojac P, Rousseau F, Pal LR, Savojardo C, Schymkowitz J, Thusberg J, Tosatto SCE, Vihinen M, Valiaho J, Repo S, Moult J, Brenner SE, **Friedberg I**
Assessing computational predictions of the phenotypic effect of cystathionine-beta-synthase variants.
Hum Mutat. 2019 Sep;40(9):1530-1545. [doi](https://dx.doi.org/10.1002/humu.23868. Epub 2019 Sep 3)


+ Kacsoh BZ, Barton S, Jiang Y, Zhou N, Mooney SD, **Friedberg I**, Radivojac P, Greene CS, Bosco G
New Drosophila Long-Term Memory Genes Revealed by Assessing Computational Function Prediction Methods.
G3 (Bethesda). 2019 Jan 9;9(1):251-267. [doi](https://dx.doi.org/10.1534/g3.118.200867)


+ Hu X, **Friedberg I**
SwiftOrtho: A fast, memory-efficient, multiple genome orthology classifier.
Gigascience. 2019 Oct 1;8(10). pii: 5606727. [doi](https://dx.doi.org/10.1093/gigascience/giz118)


+ Hamid MN, **Friedberg I**
Identifying antimicrobial peptides using word embedding with deep recurrent neural networks.
Bioinformatics. 2019 Jun 1;35(12):2009-2016. [doi](https://dx.doi.org/10.1093/bioinformatics/bty937)


+ Cho KT, Portwood JL 2nd, Gardiner JM, Harper LC, Lawrence-Dill CJ, **Friedberg I**, Andorf CM
MaizeDIG: Maize Database of Images and Genomes.
Front Plant Sci. 2019 Aug 28;10:1050. [doi](https://dx.doi.org/10.3389/fpls.2019.01050)


+ Cao L, Gurevich A, Alexander KL, Naman CB, Leao T, Glukhov E, Luzzatto-Knaan T, Vargas F, Quinn R, Bouslimani A, Nothias LF, Singh NK, Sanders JG, Benitez RAS, Thompson LR, Hamid MN, Morton JT, Mikheenko A, Shlemov A, Korobeynikov A, **Friedberg I**, Knight R, Venkateswaran K, Gerwick WH, Gerwick L, Dorrestein PC, Pevzner PA, Mohimani H
MetaMiner: A Scalable Peptidogenomics Approach for Discovery of Ribosomal Peptide Natural Products with Blind Modifications from Microbial Communities.
Cell Syst. 2019 Dec 18;9(6):600-608.e4. [doi](https://dx.doi.org/10.1016/j.cels.2019.09.004)


+ Zhou N, Siegel ZD, Zarecor S, Lee N, Campbell DA, Andorf CM, Nettleton D, Lawrence-Dill CJ, Ganapathysubramanian B, Kelly JW, **Friedberg I**
Crowdsourcing image analysis for plant phenomics to generate ground truth data for machine learning.
PLoS Comput Biol. 2018 Jul 30;14(7):e1006337. [doi](https://dx.doi.org/10.1371/journal.pcbi.1006337)


+ Wimalanathan K, **Friedberg I**, Andorf CM, Lawrence-Dill CJ
Maize GO Annotation-Methods, Evaluation, and Review (maize-GAMER).
Plant Direct. 2018 Apr 11;2(4):e00052. [doi](https://dx.doi.org/10.1002/pld3.52)


+ **Friedberg I**, Radivojac P
Community-Wide Evaluation of Computational Function Prediction.
Methods Mol Biol. 2017;1446:133-146. [doi](https://dx.doi.org/10.1007/978-1-4939-3743-1_10)



+ Jiang Y, Oron TR, Clark WT, Bankapur AR, D'Andrea D, Lepore R, Funk CS, Kahanda I, Verspoor KM, Ben-Hur A, Koo da CE, Penfold-Brown D, Shasha D, Youngs N, Bonneau R, Lin A, Sahraeian SM, Martelli PL, Profiti G, Casadio R, Cao R, Zhong Z, Cheng J, Altenhoff A, Skunca N, Dessimoz C, Dogan T, Hakala K, Kaewphan S, Mehryary F, Salakoski T, Ginter F, Fang H, Smithers B, Oates M, Gough J, Toronen P, Koskinen P, Holm L, Chen CT, Hsu WL, Bryson K, Cozzetto D, Minneci F, Jones DT, Chapman S, Bkc D, Khan IK, Kihara D, Ofer D, Rappoport N, Stern A, Cibrian-Uhalte E, Denny P, Foulger RE, Hieta R, Legge D, Lovering RC, Magrane M, Melidoni AN, Mutowo-Meullenet P, Pichler K, Shypitsyna A, Li B, Zakeri P, ElShal S, Tranchevent LC, Das S, Dawson NL, Lee D, Lees JG, Sillitoe I, Bhat P, Nepusz T, Romero AE, Sasidharan R, Yang H, Paccanaro A, Gillis J, Sedeno-Cortes AE, Pavlidis P, Feng S, Cejuela JM, Goldberg T, Hamp T, Richter L, Salamov A, Gabaldon T, Marcet-Houben M, Supek F, Gong Q, Ning W, Zhou Y, Tian W, Falda M, Fontana P, Lavezzo E, Toppo S, Ferrari C, Giollo M, Piovesan D, Tosatto SC, Del Pozo A, Fernandez JM, Maietta P, Valencia A, Tress ML, Benso A, Di Carlo S, Politano G, Savino A, Rehman HU, Re M, Mesiti M, Valentini G, Bargsten JW, van Dijk AD, Gemovic B, Glisic S, Perovic V, Veljkovic V, Veljkovic N, Almeida-E-Silva DC, Vencio RZ, Sharan M, Vogel J, Kansakar L, Zhang S, Vucetic S, Wang Z, Sternberg MJ, Wass MN, Huntley RP, Martin MJ, O'Donovan C, Robinson PN, Moreau Y, Tramontano A, Babbitt PC, Brenner SE, Linial M, Orengo CA, Rost B, Greene CS, Mooney SD, **Friedberg I**, Radivojac P
An expanded evaluation of protein function prediction methods shows an improvement in accuracy.
Genome Biol. 2016 Sep 7;17(1):184. [doi](https://dx.doi.org/10.1186/s13059-016-1037-6)


+ Cigarroa-Toledo N, Blitvich BJ, Cetina-Trejo RC, Talavera-Aguilar LG, Baak-Baak CM, Torres-Chable OM, Hamid MN, **Friedberg I**, Gonzalez-Martinez P, Alonzo-Salomon G, Rosado-Paredes EP, Rivero-Cardenas N, Reyes-Solis GC, Farfan-Ale JA, Garcia-Rejon JE, Machain-Williams C
Chikungunya Virus in Febrile Humans and Aedes aegypti Mosquitoes, Yucatan, Mexico.
Emerg Infect Dis. 2016 Oct;22(10):1804-7. [doi](https://dx.doi.org/10.3201/eid2210.152087)


+ Ream DC, Bankapur AR, **Friedberg I**
An event-driven approach for studying gene block evolution in bacteria.
Bioinformatics. 2015 Jul 1;31(13):2075-83. [doi](https://dx.doi.org/10.1093/bioinformatics/btv128)


+ Morton JT, Freed SD, Lee SW, **Friedberg I**
A large scale prediction of bacteriocin gene blocks suggests a wide functional spectrum for bacteriocins.
BMC Bioinformatics. 2015 Nov 11;16:381. [doi](https://dx.doi.org/10.1186/s12859-015-0792-9)


+ **Friedberg I**, Wass MN, Mooney SD, Radivojac P
Ten simple rules for a community computational challenge.
PLoS Comput Biol. 2015 Apr 23;11(4):e1004150. [doi](https://dx.doi.org/10.1371/journal.pcbi.1004150)


+ Wass MN, Mooney SD, Linial M, Radivojac P, **Friedberg I**
The automated function prediction SIG looks back at 2013 and prepares for 2014.
Bioinformatics. 2014 Jul 15;30(14):2091-2. [doi](https://dx.doi.org/10.1093/bioinformatics/btu117)


+ Jiang Y, Clark WT, **Friedberg I**, Radivojac P
The impact of incomplete knowledge on the evaluation of protein function prediction: a structured-output learning perspective.
Bioinformatics. 2014 Sep 1;30(17):i609-16. [doi](https://dx.doi.org/10.1093/bioinformatics/btu472)


+ Schnoes AM, Ream DC, Thorman AW, Babbitt PC, **Friedberg I**
Biases in the experimental annotations of protein function and their effect on our understanding of protein function space.
PLoS Comput Biol. 2013;9(5):e1003063. [doi](https://dx.doi.org/10.1371/journal.pcbi.1003063)


+ Ream DC, Murakami ST, Schmidt EE, Huang GH, Liang C, **Friedberg I**, Cheng XW
Comparative analysis of error-prone replication mononucleotide repeats across baculovirus genomes.
Virus Res. 2013 Dec 26;178(2):217-25. [doi](https://dx.doi.org/10.1016/j.virusres.2013.10.005)


+ Radivojac P, Clark WT, Oron TR, Schnoes AM, Wittkop T, Sokolov A, Graim K, Funk C, Verspoor K, Ben-Hur A, Pandey G, Yunes JM, Talwalkar AS, Repo S, Souza ML, Piovesan D, Casadio R, Wang Z, Cheng J, Fang H, Gough J, Koskinen P, Toronen P, Nokso-Koivisto J, Holm L, Cozzetto D, Buchan DW, Bryson K, Jones DT, Limaye B, Inamdar H, Datta A, Manjari SK, Joshi R, Chitale M, Kihara D, Lisewski AM, Erdin S, Venner E, Lichtarge O, Rentzsch R, Yang H, Romero AE, Bhat P, Paccanaro A, Hamp T, Kassner R, Seemayer S, Vicedo E, Schaefer C, Achten D, Auer F, Boehm A, Braun T, Hecht M, Heron M, Honigschmid P, Hopf TA, Kaufmann S, Kiening M, Krompass D, Landerer C, Mahlich Y, Roos M, Bjorne J, Salakoski T, Wong A, Shatkay H, Gatzmann F, Sommer I, Wass MN, Sternberg MJ, Skunca N, Supek F, Bosnjak M, Panov P, Dzeroski S, Smuc T, Kourmpetis YA, van Dijk AD, ter Braak CJ, Zhou Y, Gong Q, Dong X, Tian W, Falda M, Fontana P, Lavezzo E, Di Camillo B, Toppo S, Lan L, Djuric N, Guo Y, Vucetic S, Bairoch A, Linial M, Babbitt PC, Brenner SE, Orengo C, Rost B, Mooney SD, **Friedberg I**
A large-scale evaluation of computational protein function prediction.
Nat Methods. 2013 Mar;10(3):221-7. [doi](https://dx.doi.org/10.1038/nmeth.2340)


+ Oberlin AT, Jurkovic DA, Balish MF, **Friedberg I**
Biological database of images and genomes: tools for community annotations linking image and genomic information.
Database (Oxford). 2013 Apr 2;2013:bat016. [doi](https://dx.doi.org/10.1093/database/bat016)


+ Anton BP, Chang YC, Brown P, Choi HP, Faller LL, Guleria J, Hu Z, Klitgord N, Levy-Moonshine A, Maksad A, Mazumdar V, McGettrick M, Osmani L, Pokrzywa R, Rachlin J, Swaminathan R, Allen B, Housman G, Monahan C, Rochussen K, Tao K, Bhagwat AS, Brenner SE, Columbus L, de Crecy-Lagard V, Ferguson D, Fomenkov A, Gadda G, Morgan RD, Osterman AL, Rodionov DA, Rodionova IA, Rudd KE, Soll D, Spain J, Xu SY, Bateman A, Blumenthal RM, Bollinger JM, Chang WS, Ferrer M, **Friedberg I**, Galperin MY, Gobeill J, Haft D, Hunt J, Karp P, Klimke W, Krebs C, Macelis D, Madupu R, Martin MJ, Miller JH, O'Donovan C, Palsson B, Ruch P, Setterdahl A, Sutton G, Tate J, Yakunin A, Tchigvintsev D, Plata G, Hu J, Greiner R, Horn D, Sjolander K, Salzberg SL, Vitkup D, Letovsky S, Segre D, DeLisi C, Roberts RJ, Steffen M, Kasif S
The COMBREX project: design, methodology, and initial results.
PLoS Biol. 2013;11(8):e1001638. [doi](https://dx.doi.org/10.1371/journal.pbio.1001638)

+ Schwartz S, **Friedberg I**, Ivanov IV, Davidson LA, Goldsby JS, Dahl DB, Herman D, Wang M, Donovan SM, Chapkin RS
A metagenomic study of diet-dependent interaction between gut microbiota and host in infants reveals differences in immune response.
Genome Biol. 2012 Apr 30;13(4):r32. [doi](https://dx.doi.org/10.1186/gb-2012-13-4-r32)


+ Donovan SM, Wang M, Li M, **Friedberg I**, Schwartz SL, Chapkin RS
Host-microbe interactions in the neonatal intestine: role of human milk oligosaccharides.
Adv Nutr. 2012 May 1;3(3):450S-5S. [doi](https://dx.doi.org/10.3945/an.112.001859)


+ Bielewicz S, Bell E, Kong W, **Friedberg I**, Priscu JC, Morgan-Kiss RM
Protist diversity in a permanently ice-covered Antarctic lake during the polar night transition.
ISME J. 2011 Sep;5(9):1559-64. [doi](https://dx.doi.org/10.1038/ismej.2011.23)


+ Wooley JC, Godzik A, **Friedberg I**
A primer on metagenomics.
PLoS Comput Biol. 2010 Feb 26;6(2):e1000667. [doi](https://dx.doi.org/10.1371/journal.pcbi.1000667)


+ Kelly RJ, Vincent DE, **Friedberg I**
IPRStats: visualization of the functional potential of an InterProScan run.
BMC Bioinformatics. 2010 Dec 21;11 Suppl 12:S13. [doi](https://dx.doi.org/10.1186/1471-2105-11-S12-S13)


+ Field D, Sansone S, Delong EF, Sterk P, **Friedberg I**, Kottmann R, Hirschman L, Garrity G, Cochrane G, Wooley J, Meyer F, Hunter S, White O
Meeting Report: Metagenomics, Metadata and MetaAnalysis (M3) at ISMB 2010.
Stand Genomic Sci. 2010 Dec 4;3(3):232-4. [doi](https://dx.doi.org/10.4056/sigs.1383476)


+ Field D, Sansone S, Delong EF, Sterk P, **Friedberg I**, Gaudet P, Lewis S, Kottmann R, Hirschman L, Garrity G, Cochrane G, Wooley J, Meyer F, Hunter S, White O, Bramlett B, Gregurick S, Lapp H, Orchard S, Rocca-Serra P, Ruttenberg A, Shah N, Taylor C, Thessen A
Meeting Report: BioSharing at ISMB 2010.
Stand Genomic Sci. 2010 Dec 4;3(3):254-8. [doi](https://dx.doi.org/10.4056/sigs/1403501)


+ Field D, **Friedberg I**, Sterk P, Kottmann R, Glockner FO, Hirschman L, Garrity GM, Cochrane G, Wooley J, Gilbert J
Meeting Report: "Metagenomics, Metadata and Meta-analysis" (M3) Special Interest Group at ISMB 2009.
Stand Genomic Sci. 2009 Dec 29;1(3):278-82. [doi](https://dx.doi.org/10.4056/sigs.641096)


+ Cock PJ, Antao T, Chang JT, Chapman BA, Cox CJ, Dalke A, **Friedberg I**, Hamelryck T, Kauff F, Wilczynski B, de Hoon MJ
Biopython: freely available Python tools for computational molecular biology and bioinformatics.
Bioinformatics. 2009 Jun 1;25(11):1422-3. [doi](https://dx.doi.org/10.1093/bioinformatics/btp163)


+ Rodrigues AP, Grant BJ, Godzik A, **Friedberg I**
The 2006 automated function prediction meeting.
BMC Bioinformatics. 2007 May 22;8 Suppl 4:S1-4. [doi](https://dx.doi.org/10.1186/1471-2105-8-S4-S1)


+ **Friedberg I**, Nika K, Tautz L, Saito K, Cerignoli F, **Friedberg I**, Godzik A, Mustelin T
Identification and characterization of DUSP27, a novel dual-specific protein phosphatase.
FEBS Lett. 2007 May 29;581(13):2527-33. [doi](https://dx.doi.org/10.1016/j.febslet.2007.04.059)


+ **Friedberg I**, Harder T, Kolodny R, Sitbon E, Li Z, Godzik A
Using an alignment of fragment strings for comparing protein structures.
Bioinformatics. 2007 Jan 15;23(2):e219-24. [doi](https://dx.doi.org/10.1093/bioinformatics/btl310)


+ **Friedberg I**, Godzik A
Functional differentiation of proteins: implications for structural genomics.
Structure. 2007 Apr;15(4):405-15. [doi](https://dx.doi.org/10.1016/j.str.2007.02.005)


+ **Friedberg I**, Jambon M, Godzik A
New avenues in protein function prediction.
Protein Sci. 2006 Jun;15(6):1527-9. [doi](https://dx.doi.org/10.1110/ps.062158406)


+ **Friedberg I**, Harder T, Godzik A
JAFA: a protein function annotation meta-server.
Nucleic Acids Res. 2006 Jul 1;34(Web Server issue):W379-81. [doi](https://dx.doi.org/10.1093/nar/gkl045)


+ **Friedberg I**
Automated protein function prediction--the genomic challenge.
Brief Bioinform. 2006 Sep;7(3):225-42. [doi](https://dx.doi.org/10.1093/bib/bbl004)


+ Bourne PE, **Friedberg I**
Ten simple rules for selecting a postdoctoral position.
PLoS Comput Biol. 2006 Nov 24;2(11):e121. [doi](https://dx.doi.org/10.1371/journal.pcbi.0020121)


+ **Friedberg I**, Godzik A
Fragnostic: walking through protein structure space.
Nucleic Acids Res. 2005 Jul 1;33(Web Server issue):W249-51. [doi](https://dx.doi.org/10.1093/nar/gki363)


+ **Friedberg I**, Godzik A
Connecting the protein structure universe by using sparse recurring fragments.
Structure. 2005 Aug;13(8):1213-24. [doi](https://dx.doi.org/10.1016/j.str.2005.05.009)


+ **Friedberg I**, Jaroszewski L, Ye Y, Godzik A
The interplay of fold recognition and experimental structure determination in structural genomics.
Curr Opin Struct Biol. 2004 Jun;14(3):307-12. [doi](https://dx.doi.org/10.1016/j.sbi.2004.04.005)


+ Alonso A, Sasin J, Bottini N, Friedberg I, **Friedberg I**, Osterman A, Godzik A, Hunter T, Dixon J, Mustelin T
Protein tyrosine phosphatases in the human genome.
Cell. 2004 Jun 11;117(6):699-711. [doi](https://dx.doi.org/10.1016/j.cell.2004.05.018)


+ **Friedberg I**, Margalit H
Persistently conserved positions in structurally similar, sequence dissimilar proteins: roles in preserving protein fold and function.
Protein Sci. 2002 Feb;11(2):350-60. [doi](https://dx.doi.org/10.1110/ps.18602)


+ **Friedberg I**, Margalit H
PeCoP: automatic determination of persistently conserved positions in protein families.
Bioinformatics. 2002 Sep;18(9):1276-7. [doi](https://dx.doi.org/10.1093/bioinformatics/18.9.1276)

+ **Friedberg I**, Kaplan T, Margalit H
Glimmers in the Midnight Zone: Characterization of Aligned Identical Residues in Sequence-Dissimilar Proteins Sharing a Common Fold
Proceedings, Eighth International Conference on Intelligent Systems for Molecular Biology (2000) 

+ N Cherradi, M F Rossier, M B Vallotton, R Timberg, **I Friedberg**, J Orly, X J Wang, D M Stocco, A M Capponi
Submitochondrial distribution of three key steroidogenic proteins (steroidogenic acute regulatory protein and cytochrome p450scc and 3beta-hydroxysteroid dehydrogenase isomerase enzymes) upon stimulation by intracellular calcium in adrenal glomerulosa cells 
J Biol Chem (1997) Mar 21;272(12):7899-907 [doi](https://doi.org/10.1074/jbc.272.12.7899)




